pytx/pytx/vocabulary.py

```diff
@@ -5,7 +5,7 @@ class ThreatExchange(object):
     """
 
     URL = 'https://graph.facebook.com/'
-    VERSION = 'v2.6/'
+    VERSION = 'v2.8/'
```

Comment:
Correct. When they go to POST they'll get whatever the response is from TX (success or failure). Since Descriptors are returning those fields still in `2.8` it's necessary to keep the class attributes so there's no failures building the object.

I also figured leaving the vocabulary would be useful to keep. Even if it's not tied to a specific object, they might be good reference material for tags.